src/tablestore.rs

```diff
@@ -483,6 +569,27 @@ mod tests {
 
     const ROOT: &str = "/root";
 
+    #[tokio::test]
+    async fn test_parse_id() {
```

Comment:
This test could be synchronous:

```rust
#[test]
fn test_parse_id() {
```

If you want to use `async` tests across the board for consistency, I'm fine with that though.